app/app.go

```diff
@@ -1003,10 +1003,9 @@ func NewApp(
 
 	// Register SEDA signer and ExtendVote handler.
 	pvKeyFile := filepath.Join(homePath, cast.ToString(appOpts.Get("priv_validator_key_file")))
-	loadPath := filepath.Join(filepath.Dir(pvKeyFile), utils.SEDAKeyFileName)
-	signer, err := utils.LoadSEDASigner(loadPath)
+	signer, err := utils.LoadSEDASigner(pvKeyFile)
```

Comment:
Stupid idea time!

Given that we're expecting validators to create the keyfile with a transaction from the validator operator account, can we not derive the operator address at that point in time and store it in the keyfile? That way we're less dependent on the priv key file and we don't have to do a lookup by consensus address. Also reduces the risk of issues when we upgrade the SDK to a version that has consensus key rotation.

Reply:
if this is your stupid idea, how good are your smart ideas? 😱

Reply:
addressed in https://github.com/sedaprotocol/seda-chain/pull/413/commits/897dd27f33dc19e38fc9b5e96f5ed828623289ce